packages/MarkdownEditor-Core.package/MarkdownCommentBlock.class/class/matches..st

```diff
@@ -0,0 +1,7 @@
+testing
+matches: aString
+	
+	^ (aString beginsWith: '    ') not
+		and: [aString withBlanksTrimmed beginsWith: '<!--']
+		and: [((aString includesSubstring: '-->') not)
+			or: [(aString withBlanksTrimmed findString: '-->') >= 5]]
```

Comment:
Can we avoid the magic number here? 🤔 Or do you think it is for everyone clear?

Reply:
We could say '<!--' size +1, as that is where the number comes from. The starting position of the closing tag may not be at the first 4 characters, if it is in the same line as the opening tag. Would that be better?

Reply:
This makes me want to introduce 
```
MarkdownCommentBlock>>openingTag

    ^ '<!--'
```
on the class side and the same for `MarkdownCommentBlock>>closingTag`. Then we could use that and ask for the size and then the index shifting would make sense. We could also use those in `matches:` and `isClosed`. That way we could get rid of the magic numbers and magic strings at the same time.

Obviously this is not quite simple design anymore and the argument of changing magic strings doesn't really apply because we don't really expect the Markdown and/or HTML spec to change but on the software engineering side of things it would be an improvement I think.

Reply:
I would say that `openingTag` is in the spirit of simple design: Make it clear what you are doing, by expressing it with the least possible _complexity_. (And number of methods is only a hint to complexity, not a hard criterium)